### ISSUE TITLE
Warning messages 'Overwriting existing entry for backend'

### ISSUE THREAD

**Issue by wheestermans31** [F5Networks/k8s-bigip-ctlr#1077]
#### Description

Inside our F5-BIGIP controller logs in kubernetes we see a lot of overwrite warning messages:
```
2019/11/07 15:07:43 [WARNING] Overwriting existing entry for backend {ServiceName:tme-svc-http-cwsweb-ui ServicePort:8080 Namespace:tme-ns-http-dev}
2019/11/07 15:07:43 [WARNING] Overwriting existing entry for backend {ServiceName:tme-svc-http-cwsweb-ui ServicePort:8080 Namespace:tme-ns-http-dev}
2019/11/07 15:07:43 [WARNING] Overwriting existing entry for backend {ServiceName:tme-svc-http-cwsweb-ui ServicePort:8080 Namespace:tme-ns-http-dev}
2019/11/07 15:07:43 [WARNING] Overwriting existing entry for backend {ServiceName:tme-svc-http-cwsweb-ui ServicePort:8080 Namespace:tme-ns-http-dev}
2019/11/07 15:07:46 [WARNING] Overwriting existing entry for backend {ServiceName:tme-svc-jetty-vlps-hub-service-app-se ServicePort:8080 Namespace:tme-ns-jetty-dev}
2019/11/07 15:07:46 [WARNING] Overwriting existing entry for backend {ServiceName:tme-svc-jetty-vlps-hub-service-app-se ServicePort:8080 Namespace:tme-ns-jetty-dev}
2019/11/07 15:07:46 [WARNING] Overwriting existing entry for backend {ServiceName:tme-svc-jetty-vlps-hub-service-app-se ServicePort:8080 Namespace:tme-ns-jetty-dev}
2019/11/07 15:07:46 [WARNING] Overwriting existing entry for backend {ServiceName:tme-svc-jetty-vlps-hub-service-app-se ServicePort:8080 Namespace:tme-ns-jetty-dev}
2019/11/07 15:07:47 [WARNING] Overwriting existing entry for backend {ServiceName:tme-svc-jetty-vlps-hub-service-app-se ServicePort:8080 Namespace:tme-ns-jetty-dev}
```
I see even messages for non-changed resources. What do these warnings mean? Can we just ignore? 

#### Kubernetes Version
1.11.1 (Docker EE 2.1)

#### Controller Version
1.11.1

**Comment by wheestermanstoyota** [comment]
Any update on the question?

Regards
Walter

**Comment by wheestermanstoyota** [comment]
I find it very weird that I see so many lines for the same in 1 second?
...
2019/11/15 12:45:36 [WARNING] Overwriting existing entry for backend {ServiceName:tme-svc-jetty-a2dweb ServicePort:8080 Namespace:tme-ns-jetty-dev}
2019/11/15 12:45:36 [WARNING] Overwriting existing entry for backend {ServiceName:tme-svc-jetty-a2dweb ServicePort:8080 Namespace:tme-ns-jetty-dev}
2019/11/15 12:45:36 [WARNING] Overwriting existing entry for backend {ServiceName:tme-svc-jetty-a2dweb ServicePort:8080 Namespace:tme-ns-jetty-dev}
2019/11/15 12:45:36 [WARNING] Overwriting existing entry for backend {ServiceName:tme-svc-jetty-a2dweb ServicePort:8080 Namespace:tme-ns-jetty-dev}
2019/11/15 12:45:36 [WARNING] Overwriting existing entry for backend {ServiceName:tme-svc-jetty-a2dweb ServicePort:8080 Namespace:tme-ns-jetty-dev}
2019/11/15 12:45:37 [WARNING] Overwriting existing entry for backend {ServiceName:tme-svc-jetty-a2dweb ServicePort:8080 Namespace:tme-ns-jetty-dev}
2019/11/15 12:45:37 [WARNING] Overwriting existing entry for backend {ServiceName:tme-svc-jetty-a2dweb ServicePort:8080 Namespace:tme-ns-jetty-dev}
2019/11/15 12:45:37 [WARNING] Overwriting existing entry for backend {ServiceName:tme-svc-jetty-a2dweb ServicePort:8080 Namespace:tme-ns-jetty-dev}
2019/11/15 12:45:37 [WARNING] Overwriting existing entry for backend {ServiceName:tme-svc-jetty-a2dweb ServicePort:8080 Namespace:tme-ns-jetty-dev}
2019/11/15 12:45:37 [WARNING] Overwriting existing entry for backend {ServiceName:tme-svc-jetty-a2dweb ServicePort:8080 Namespace:tme-ns-jetty-dev}
2019/11/15 12:45:38 [WARNING] Overwriting existing entry for backend {ServiceName:tme-svc-jetty-a2dweb ServicePort:8080 Namespace:tme-ns-jetty-dev}
2019/11/15 12:45:38 [WARNING] Overwriting existing entry for backend {ServiceName:tme-svc-jetty-a2dweb ServicePort:8080 Namespace:tme-ns-jetty-dev}
2019/11/15 12:45:38 [WARNING] Overwriting existing entry for backend {ServiceName:tme-svc-jetty-a2dweb ServicePort:8080 Namespace:tme-ns-jetty-dev}
2019/11/15 12:45:38 [WARNING] Overwriting existing entry for backend {ServiceName:tme-svc-jetty-a2dweb ServicePort:8080 Namespace:tme-ns-jetty-dev}
2019/11/15 12:45:38 [WARNING] Overwriting existing entry for backend {ServiceName:tme-svc-jetty-a2dweb ServicePort:8080 Namespace:tme-ns-jetty-dev}
2019/11/15 12:45:39 [WARNING] Overwriting existing entry for backend {ServiceName:tme-svc-jetty-a2dweb ServicePort:8080 Namespace:tme-ns-jetty-dev}
2019/11/15 12:45:39 [WARNING] Overwriting existing entry for backend {ServiceName:tme-svc-jetty-a2dweb ServicePort:8080 Namespace:tme-ns-jetty-dev}
2019/11/15 12:45:39 [WARNING] Overwriting existing entry for backend {ServiceName:tme-svc-jetty-a2dweb ServicePort:8080 Namespace:tme-ns-jetty-dev}
2019/11/15 12:45:39 [WARNING] Overwriting existing entry for backend {ServiceName:tme-svc-jetty-a2dweb ServicePort:8080 Namespace:tme-ns-jetty-dev}
2019/11/15 12:45:39 [WARNING] Overwriting existing entry for backend {ServiceName:tme-svc-jetty-a2dweb ServicePort:8080 Namespace:tme-ns-jetty-dev}
2019/11/15 12:45:40 [WARNING] Overwriting existing entry for backend {ServiceName:tme-svc-jetty-a2dweb ServicePort:8080 Namespace:tme-ns-jetty-dev}
2019/11/15 12:45:40 [WARNING] Overwriting existing entry for backend {ServiceName:tme-svc-jetty-a2dweb ServicePort:8080 Namespace:tme-ns-jetty-dev}
2019/11/15 12:45:40 [WARNING] Overwriting existing entry for backend {ServiceName:tme-svc-jetty-a2dweb ServicePort:8080 Namespace:tme-ns-jetty-dev}
2019/11/15 12:45:40 [WARNING] Overwriting existing entry for backend {ServiceName:tme-svc-jetty-a2dweb ServicePort:8080 Namespace:tme-ns-jetty-dev}
2019/11/15 12:45:40 [WARNING] Overwriting existing entry for backend {ServiceName:tme-svc-jetty-a2dweb ServicePort:8080 Namespace:tme-ns-jetty-dev}

...

**Comment by wheestermans31** [comment]
Any update?

**Comment by ssurenr** [comment]
Marking for a fix in 1.13

**Comment by ssurenr** [comment]
Internal tracking CONTCNTR-1404

**Comment by hunterx19** [comment]
I have the same problem in CIS 1.14
Is this fixed?

**Comment by mdditt2000** [comment]
@hunterx19 issue is still in CIS 1.14. Issue is resolved in CIS 2.0

**Comment by mdditt2000** [comment]
Issue is resolved in CIS 2.0. Closing this Github issue out.
